src/lib/todos/todoDataSlice.ts

```diff
@@ -0,0 +1,46 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { SelectTodo } from '@/db/schema/todos';
+
+interface TodoDataState {
+  sessionId: number | undefined;
+  displayingDate: Date;
+  todoId: number;
+  todoListData: SelectTodo[];
+  isLoading: boolean;
+}
+
+const initialState: TodoDataState = {
+  sessionId: undefined,
+  displayingDate: new Date(),
```

Comment:
p3: 새로고침 할 때, 현재 날짜가 보이게 되는 이슈는 아마 여기서 초기 값이 현재 date로 되어서 그런 것으로 파악했습니다.
```suggestion
  displayingDate: new Date('2024-08-16'), 
```
이런식으로 수정해보니 그럼 새로고침 시 값이 새로 적재되기 전16일이 뜨게 됩니다. 초기 값을 오늘 날이 아니라, null이나 undefined 값을 주고, url에 있는 값을 적재하기 전까지 falsy한 값일 때, 스캘레톤 등을 보여주는 방법은 어떤가요?

그렇게 된다면 현재 날이 보였다가 url에 있는 올바른 데이터 값으로 변하는 것은 막을 수 있지 않을까 싶네요..!

Reply:
오 스켈레톤 만드는 방법 좋은 것 같습니다! 감사합니다!